### ISSUE TITLE
Improve the style variations control accessibility

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#11044]
Reference:

WCAG Success Criterion 2.5.3 Label in Name
https://www.w3.org/TR/WCAG21/#label-in-name

> For user interface components with labels that include text or images of text, the name contains the text that is presented visually.
> A best practice is to have the text of the label at the start of the name.

Understanding Success Criterion 2.5.3: Label in Name
https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html

For example, one of the quote style visual label is:
`Regular`

The associated `aria-label` is:
`Apply style variation "Regular"`

<img width="381" alt="screen shot 2018-10-25 at 10 38 36" src="https://user-images.githubusercontent.com/1682452/47487139-2bdac980-d842-11e8-846b-502a578b5bcb.png">

Speech input users will _see_ a visible label `Regular` and will try to voice a command "Click Regular" but this will fail as the accessible name of the control is different.

As a best practice, and to meet WCAG, the accessible name should match (or at least start with) the visible label.
